### DISCUSSION TITLE
nuxt3 compatibility

### DISCUSSION THREAD

**Discussion by silkentrance** [nuxt/content#1023] [General]
I was wondering whether there is a roadmap for `@nuxt/content` for `nuxt3` support, yet?

The reason I am asking is that I am currently trying to set up a project using `nuxt3` and `@nuxt/content`, but it will fail on undefined symbols, which one could expect due to the major version change.

**Comment by shellen** [comment]
https://github.com/nuxt/content/issues/974#issuecomment-972775838

**Comment by benoitdemaegdt** [comment]
There may be a version ready for end of April 🤞 

https://github.com/nuxt/framework/pull/4093

**Comment by lucpotage** [comment]
In case you missed it, version 2 was released 2 days ago: https://content.nuxtjs.org/blog/announcing-v2
